src/python/test/test_simplex_tree.py

```diff
@@ -11,6 +11,7 @@
 from gudhi import SimplexTree, __GUDHI_USE_EIGEN3
```

Comment:
Did you use `black` to format it ? I like it ;-)

Reply:
Yes. I thought it was already formatted with black and it would just reformat the array for me, and when I realized it had modified the whole file... Well, too late.